harness/blastn/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ncbi/blast AS blast
```

Comment:
You don't want to name the stage unless there are other stages (`FROM` invocations).

Reply:
Oops, forgot to delete